samples/action_scripts/commands.json

```
@@ -318,5 +318,16 @@
 							"/f"
 						]
 					]
-		}
+		},
+
+	"INSTALL_ACTIVE_DIRECTORY_DOMAIN_SERVICES": {
+		"TYPE": "SCRIPT",
+		"TODO": "XXX: Confirm if relative paths for FILENAME work here on CI: ./action_scripts/install_active_directory_forest.ps1",
+		"FILENAME": "/r7-source/vm-automation/samples/action_scripts/install_active_directory_forest.ps1",
```

I could be wrong; but the file upload logic looked like it needed absolute paths to work in its current implementation. Will update this line/the code after it's confirmed if that's the case or not

---

I believe the expectation is that the `manageServices.py` will be run from the `samples` directory so we should be able to go with:

```suggestion
		"FILENAME": "./action_scripts/install_active_directory_forest.ps1",
```

I would say it is reasonable to keep that expectation in place for now as it matches other uses of `FILENAME` in this config file.